package-lock.json

```diff
@@ -4415,554 +4414,6 @@
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8=",
       "dev": true
     },
-    "fsevents": {
```

Comment:
Isn't this just some difference between macOS and Linux? https://github.com/npm/npm/issues/17722

Reply:
Ah, that is right, I completely forgot about this. Thanks for letting me know!